### PR TITLE
Update installation_win.rst

### PR DESCRIPTION
**Pull request by yerpj**
Step 6, `pip install --user -r scripts/requirements.txt` path for `scripts` directory is actually `zephyr/scripts`.

### CODE REVIEW

File: doc/getting_started/installation_win.rst [hunk 1]
```diff
@@ -73,7 +73,7 @@ environment for Windows. Follow the steps below to set it up:
       $ curl -O 'https://bootstrap.pypa.io/get-pip.py'
       $ ./get-pip.py
       $ rm get-pip.py
-      $ pip install --user -r scripts/requirements.txt
+      $ pip install --user -r zephyr/scripts/requirements.txt
```
Comment (dbkinder) [inline doc/getting_started/installation_win.rst]: No. We make no assumption of the root directory name where the Zephyr source is cloned.

**Review by dbkinder — CHANGES_REQUESTED** [review]
Path should remain as scripts/

File: doc/getting_started/installation_win.rst (hunk 1 above)
Comment (yerpj) [reply doc/getting_started/installation_win.rst]: At least it is not obvious that the `scripts` directory is owned by `zephyr`. One can be confused by the current command. Maybe it should be stated that the directory is actually the installation directory of the Zephyr project. `YOUR_ZEPHYR_PROJ_INSTALL_PATH/scripts/requirements.txt`

Comment (nashif) [reply doc/getting_started/installation_win.rst]: add `$(ZEPHYR_BASE)/scripts` instead of plain 'zephyr'

Comment (dbkinder) [reply doc/getting_started/installation_win.rst]: ...as long as you've sourced the zephyr-env.sh script :)
It also indicates there's a Zephyr base directory even if you haven't sourced the script yet, so I'd go with Anas' suggestion for the edit.

**Review by dbkinder — APPROVED** [review]
As you're following the steps in the setup guide, this is indeed a valid change, so it may pass.

**Review by dbkinder — CHANGES_REQUESTED** [review]
Sigh... but there are errors in the commit body (including no Signed-off-by)

**Comment by dbkinder** [comment]
@yerpj Can you fix your commit message (``commit -s --amend`` should do the trick.)

**Comment by nashif** [comment]
another PR doing the same

**Pull request closed without merging**